Jupyter_notebook/tutorial.ipynb

```diff
@@ -0,0 +1,305 @@
kDPvbz/PnzC7+vXLlCzMPo5HDTpk2mafLUGDt2rM15yCeguXPntjlPEnx7DB06FIA+Ic/sNqQdc4w8/5iYGEl/MvJOzJw50yokACmgtxHkWiRPFfLEY7vWnM1MSH3V19dX8ltNk6e0P5fW2Q6JChUqCH/Xr18fzZs3N5yXK+KYpSGGqdhDk2cWM2fOVD2/b98+q9hXtIg9LtqbxYsXE4/L9+S5MmXLljV8rZoJptgsVAtaZzl6oNGA2QPa+zD6rhmJP2ZPjDgkSq7QPCdXMiWS11f8TsufZbt27Yh5GL2f5cuXC05+zNomIIZf2OIFJlu4evWqJASP3COg2fCCqZ5tDYmJifDy8sLRo0dRtGhRpEuXzqY60D5XI+aaf/31l+S3kbEyNjZW9zW0yH0GfPjwwTQNqqsKeTNnziRq2OxNVFQUVq9ejXHjxkmOv3v3TtHKAHDMt2bevHmYOnWq4gKXO8GEPDdEHs+KBleaYBjFFUzC3EmT56ko9eVLly5R74czws2bNzVjQNkC/+FVciVtb+Ru0N1hIcOdWLp0qbOrQIVZ5npqmO0h0l506dIFFy5csHs5T58+tbKQEId0mTdvnpWAxJOQkIAUKVKgUqVK+Pzzz6m8M6pBM+aQ4icawYj5qz3HRLm5ZtWqVTFp0iRT8nal8dQV5oP169cnHs+SJYtiXwfsN/cSOxxS8mDujjBzTTdk4sSJuq8xw5U+w7P25DmCV69eSZw/mIHa5GL48OGmliVm4sSJdjUX5j3NOctcUw7r4+bSs2dP0/Kyp9baEUKemqWAK+FIM+SGDRtKfs+aNUv4W23SKZ6w85qHzp07G64HzXOVx090pMdjs/ZFksbWAwcOSMy1IyMj7RKM3dmQ/BnExcW5jEWGmsdceagfM9iyZYuml3R3hQl5boYzJ32M/30AmSbPeaitQrrCCqVR+NALHMfh2bNnqFKlik17DWyFCXnJE9p3yJYx0F36liPHE3EIDz0kJCQI7cmHWfjjjz8M14M20Lo4Tl1SUpJVPFN7YU/B22iMOHenb9++8Pf3pw7/40zs4fxIvsDiSTBzTTfDnSexnoA77cnzVNTeAVdZibSVdevW4fDhw5L4iI6GLWQkT2jfISOeEXncpW85cpyPjo42dN358+dRvXp1AB9jsirtJ6eFD7+jl+3bt9tULi3OHBM9FT6WoThenavC5l76YJo8N0PsBY3heGbNmoXXr1+7zSTF04iLi1MV8o4dO2b3Oly+fNmu+cfHxwsr8s6E9fHkCa2Q16RJE8NluEvfslfIBBJGhWZewONp3769GdVhMFwSRwp5nmA5xzR5boZW/CZbMCtAr6ezdu1aFC1a1NnVSJb4+/sje/bszq6GXXn37p3Eo5+zcJeJOMNcjGqU9OAufSsiIsLZVXBLjh8/7lFhVRiugyNNShMTEx0Wn9JeuHftkxGPHj3CvXv37FrGxYsXFc/Nnj3brmW7E3FxcW4zSfFEbAnC7C64gqMk1seTJ3fu3LF7GY7UkDEcT1hYmLOrwPBQtm7d6rCyEhIS3F7IY+aabkKdOnUcFgicRPfu3Z1WtquxcuVKhIeHO7saDIZdYRNxhr1gfYvBYLg6jljwsjceJeTNnDkTuXLlgr+/P8qXL4+TJ086u0qm4S5xhZIDR44csYsbXwbDlbDFsQaDwWAwGO5MsWLF3H5fnscIeWvWrEHfvn0xfPhwnD17FiVLlkStWrWsAvy6K67giIHBYCQfmLaFwWAwGMkZR+xRticeI+RNnjwZnTp1Qvv27VGkSBHMmTMHAQEBWLhwobOrZjNz585FbGyss6vBYDCSEUyTx2AwGIzkzMOHD51dBZvwCCEvPj4eZ86ckeyT8vLyQnh4uENcqtuDuLg43LlzB0OHDkXXrl2dXR0Gg5HMYCbJDAaDwUjOjBw5Erdu3XJbyxb3dhvz/zx//hyJiYnInDmz5HjmzJlx/fp1q/RxcXGSBxYTE2P3Ourl5MmTTnW0wmAwGAwGg8FgJFfWr1+P9evXo3r16ti3b5+zq6Mbj9Dk6WXcuHFIly6d8C9HjhzOrpIVRYsWxV9//eXsajAYDAaDwWAwGMmOPHnyYPz48Rg1apSzq2IIj9DkZcyYEd7e3oiMjJQcj4yMRHBwsFX6wYMHo2/fvsLvmJgYlxP0goKC8OWXXyIiIgKnTp1CgwYNnF0lBoPBYDAYDAbD48mbNy9u377t7GrYhEdo8nx9fREaGoq9e/cKx5KSkrB3715iUE4/Pz+kTZtW8s9VyZw5M3Lnzu3sajAYDAaDwWAwGMkCT/Bq7xFCHgD07dsXf/zxB5YsWYJr166hW7duePv2Ldq3b+/sqtlMqlSpnF0FBsMlKF68uLOrwGAwkinp0qVzdhUYDIaD8PJyfxHJI8w1AaB58+Z49uwZhg0bhoiICJQqVQo7d+60csbijvBCXlhYGD58+IDTp087uUYMhnOYMWMGqlat6uxqMBiMZIifn5+zq8BgMBjUeIyQBwA9e/ZEz549nV0N00mdOjUAIFeuXLhz547dyvH29kZiYqLd8mcwbCVFihTOrgLDA2FjH4MGNv4wGAx3wv11kcmAlClTYvPmzZg7dy4sFovdyvH19bVb3gyGGfj4eNS6lCp58+Z1dhWSDZ5glsMwF5LTNvaN/Ei9evWcXQUGw+7Yc77tKNiXzU1o0KAB0qRJY9fJCPuASZk8ebKzq2AzGTJkcHYVTMXZK+l37951WFnXrl1zWFnJHSbkMeQEBAQgY8aMkmPOHn9cBU9wSMGw5vPPP3d2FZzOrFmz8OzZM2dXwzTYl83NsOfKgr+/v93ydkcCAwOJx3fv3o3Ro0cjf/78jq2QAa5cueLsKpiKszV5jpzcsAml43B2v2K4Jq4w2cuZMyf69++veD4kJMTUecHcuXPRqlUr0/JjMNwJi8UifHuZJo/hcOy54sw2lUtR0oKFh4fjp59+QqFChRxcI/1oCQquHD6EhLNXkB1RfsGCBTFlyhS7l8
```

Comment:
The last time I was playing with it, this step wasn't working. Can you do a little digging to see what may be going on?